data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -37,7 +37,24 @@ function release_cvo_lease {
       echo "Deleting openshift-cluster-version version cm"
       oc delete cm -n openshift-cluster-version version || sleep 5
     done
-    touch /opt/openshift/expedite_bootstrapping.done
+    touch /opt/openshift/release_cvo_lease.done
+  fi
+}
+
+function release_cpc_lease {
+  if [ ! -f /opt/openshift/release_cpc_lease.done ]
+  then
+    until [ "$(oc get leases -n kube-system cluster-policy-controller-lock | grep -c "cluster-policy-controller-lock")" -eq 0 ];
```

Comment:
seems like this part is very similar to cvo, maybe better to have a function and provide lease name and namespace?

Reply:
I tried to move the relevant parts to a common function, it seems a lot cleaner now. Thanks for the pointer @tsorya!